SPECS-EXTENDED/mdadm/mdadm.spec

```diff
@@ -1,37 +1,39 @@
-Name:        mdadm
-Version:     4.1
-#define subversion rc2
-Release:     6%{?dist}
-Summary:     The mdadm program controls Linux md devices (software RAID arrays)
+%bcond_with libreport
+Summary:        The mdadm program controls Linux md devices (software RAID arrays)
+Name:           mdadm
+Version:        4.1
+Release:        7%{?dist}
+License:        GPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-URL:         http://www.kernel.org/pub/linux/utils/raid/mdadm/
-License:     GPLv2+
-
-Source:      http://www.kernel.org/pub/linux/utils/raid/mdadm/%{name}-%{version}%{?subversion:-%{subversion}}.tar.xz
-Source1:     raid-check
-Source2:     mdadm.rules
-Source3:     mdadm-raid-check-sysconfig
-Source4:     mdmonitor.service
-Source5:     mdadm.conf
-Source6:     mdadm_event.conf
-Source7:     raid-check.timer
-Source8:     raid-check.service
-
+URL:            https://www.kernel.org/pub/linux/utils/raid/mdadm/
+Source0:        http://www.kernel.org/pub/linux/utils/raid/mdadm/%{name}-%{version}.tar.xz
+Source1:        raid-check
+Source2:        mdadm.rules
+Source3:        mdadm-raid-check-sysconfig
+Source4:        mdmonitor.service
+Source5:        mdadm.conf
+Source6:        mdadm_event.conf
+Source7:        raid-check.timer
+Source8:        raid-check.service
 # Build without -Werror.  From Debian.
-Patch00:     https://sources.debian.org/data/main/m/mdadm/4.1-2/debian/patches/debian-no-Werror.diff#/mdadm-4.1-no-Werror.patch
-
+Patch00:        https://sources.debian.org/data/main/m/mdadm/4.1-2/debian/patches/debian-no-Werror.diff#/mdadm-4.1-no-Werror.patch
 # Fedora customization patches
-Patch97:     mdadm-3.3-udev.patch
-Patch98:     mdadm-2.5.2-static.patch
-
-BuildRequires: systemd-rpm-macros binutils-devel gcc
-Requires: libreport-filesystem
-Requires(post): systemd coreutils
+Patch97:        mdadm-3.3-udev.patch
+Patch98:        mdadm-2.5.2-static.patch
+BuildRequires:  binutils-devel
+BuildRequires:  gcc
+BuildRequires:  systemd-rpm-macros
+Requires(post): coreutils
+Requires(post): systemd
+Requires(postun): coreutils
+Requires(postun): systemd
 Requires(preun): systemd
-Requires(postun): systemd coreutils
+%if %{with libreport}
+Requires:       libreport-filesystem
```

Comment:
No need for buildrequires?

Reply:
No, the ABRT/libreport functionality does not require build-time dependencies. It merely requires directories to be present at runtime.